src/components/organisms/UiMenu/stories/HasButton.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <UiPopover class="has-button">
+    <UiMenu
+      v-bind="args"
+      role="listbox"
+      :aria-label="ariaLabel"
+      aria-multiselectable="true"
+      :aria-activedescendant="ariaActivedescendant"
+      :aria-busy="ariaBusy"
+    >
+      <template #[`menu-item-${index}`] v-for="(item, index) in args.items">
+        <template v-if="item.isButton">
```

Comment:
isButton hm maybe hasCustomOption or something like that

Reply:
you should check this dependency in HasButton stories. Probably you need do something her to get this knowledge. Because CustomOption will be added here not in UiMenu.stories.ts